FlashApp.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/sh
+
+# Check and if exists, source these files /etc/include.properties and /etc/device.properties
+if [ -f /etc/include.properties ]; then
+    . /etc/include.properties
+fi
+if [ -f /etc/device.properties ]; then
+    . /etc/device.properties
+fi
+
+if [ -z "$PERSISTENT_PATH" ]; then
+    PERSISTENT_PATH="/opt"
+fi
+
+if [ -z "$LOG_PATH" ]; then
+    LOG_PATH="/opt/logs"
+    mkdir -p $LOG_PATH
+fi
+
+FLASHAPPLOGFILE="$LOG_PATH/FlashApp.log"
```

Comment:
Isnt it better to use flashapp.log instead of FlashApp.log so that it is easier to find the log instead of using capital letter

Reply:
This is temporary; logs will be added into the MW's swupdate log file or fwutility specific one when complete integration is done.